### PR TITLE
Small Fixes for Work With Project Manager Remote Templates

### PR DESCRIPTION
**Pull request by AMZN-Gene**
- Adding canonical tags.
- Fix origin field: this is now an "author" field. Now it references o3de.org as the author. 
- Fix repo.json to not use raw links (which are throttled)
- Deleting unused _origin_url_

### CODE REVIEW

File: repo.json [hunk 1]
```diff
@@ -6,6 +6,6 @@
     "additional_info": "See the README.md at the root of this repository for more information",
     "last_updated": "2022-09-27",
     "templates": [
-        "https://raw.githubusercontent.com/o3de/o3de-extras/development/Templates/Multiplayer"
+        "https://github.com/o3de/o3de-extras/tree/development/Templates/Multiplayer"
```
Comment (byrcolin) [inline repo.json:9]: not sure this is correct either, but for now ok.

**Review by byrcolin — APPROVED** [review]

File: repo.json [hunk 2]
```diff
@@ -6,6 +6,6 @@
...
     ]
 }
```
Comment (lumberyard-employee-dm) [inline repo.json:11]: nitpick: can you add a newline to this file

**Review by lumberyard-employee-dm — APPROVED** [review]

File: repo.json (hunk 1 above)
Comment (AMZN-Gene) [reply repo.json:9]: Fixed! @AMZN-Phil caught this as well. 
Updated the template link to use .git; the original returns the page you'd see in a web-browser, whereas .git causes the script to use the git API to get the raw file

File: repo.json (hunk 2 above)
Comment (AMZN-Gene) [reply repo.json:11]: fixed! thank you

**Review by AMZN-Phil — APPROVED** [review]

**Pull request merged**